src/Yoti.Auth/YotiClientEngine.cs

```diff
@@ -23,33 +23,23 @@ public YotiClientEngine(IHttpRequester httpRequester)
 #endif
         }
 
-        /// <summary>
```

Comment:
Why is the documentation going?

Reply:
We currently only have documentation for the client-facing methods. Since the methods here all come through `YotiClient`, as this is the client-facing code, then we don't need it in `YotiClientEngine` too.